src/features/events/components/EventsList/index.tsx

```diff
@@ -16,6 +16,11 @@ const sorts = {
   descending: (a: Event, b: Event) => (a.startsAt > b.startsAt ? -1 : 1),
 }
 
+const filters = {
+  future: (event: Event) => isAfter(new Date(event.startsAt), new Date()),
+  past: (event: Event) => isBefore(new Date(event.startsAt), new Date()),
+}
+
```

Comment:
This is not much different than we did with the sort functions [here](https://github.com/strvcom/frontend-academy-2022/pull/13/commits/9c15d1d6534e4c86380690e52a56b6b3269ae589#).